py-polars/tests/unit/test_df.py

```diff
@@ -1912,10 +1944,14 @@ def test_partition_by() -> None:
 def test_list_of_list_of_struct() -> None:
     expected = [{"list_of_list_of_struct": [[{"a": 1}, {"a": 2}]]}]
     pa_df = pa.Table.from_pylist(expected)
+
     df = pl.from_arrow(pa_df)
     assert df.rows() == [([[{"a": 1}, {"a": 2}]],)]
     assert df.to_dicts() == expected
 
+    df = pl.from_arrow(pa_df)[:0]
```

Comment:
Did you mean?
```python
df = pl.from_arrow(pa_df[:0])
```

Reply:
Good eyesight - you are quite right; I'll update in a subsequent push ;)